elfloader-tool/src/arch-riscv/boot.c

```diff
@@ -57,7 +59,10 @@ unsigned long l2pt[PTES_PER_PT] __attribute__((aligned(4096)));
 unsigned long l2pt_elf[PTES_PER_PT] __attribute__((aligned(4096)));
 #endif
 
-char elfloader_stack_alloc[BIT(CONFIG_KERNEL_STACK_BITS)];
+/* This stack cannot go in the .bss section because it's already in use when the
+ * .bss section is zeroed.
```

Comment:
I wonder, don't we have the same problem on ARM also then? Seem implementing `clear_bss` in assembly, so it does not use a stack is the cleanest way to avoid running into issues here.